charts/keycloak/README.md

```diff
@@ -108,6 +108,8 @@ The following table lists the configurable parameters of the Keycloak chart and
 | `ingress.tls` | TLS configuration | see below |
 | `ingress.tls[0].hosts` | List of TLS hosts | `[keycloak.example.com]` |
 | `ingress.tls[0].secretName` | Name of the TLS secret | `keycloak-tls` |
+| `networkpolicy.enabled` |If true, the ingress network policy is deployed | `false`
+| `networkpolicy.from` |Allow to define allowed external traffic (see Kubernetes doc for network policy `from` format) | ``
```

Comment:
```suggestion
| `networkpolicy.from` | Allows to define allowed external traffic (see Kubernetes doc for network policy `from` format) | ``
```

Reply:
Maybe call it `networkpolicy.extraFrom` because it might be confusing as it's optional.